package.json

```diff
@@ -19,13 +19,20 @@
 	},
 	"type": "module",
 	"bin": {
-		"listr": "cli.js"
+		"listr": "dist/cli.js"
 	},
 	"files": [
-		"cli.js"
+		"dist"
 	],
+	"engines": {
+		"node": ">=16"
+	},
 	"scripts": {
-		"test": "xo && c8 ava"
+		"prepublishOnly": "npm run build",
+		"build": "tsc && chmodx --package",
+		"postbuild": "replace-in-files dist/cli.js --string='#!/usr/bin/env tsx' --replacement='#!/usr/bin/env node'",
+		"test": "xo && tsc --noEmit && c8 yarn run:ava",
+		"run:ava": "NODE_OPTIONS='--loader=tsx --no-warnings=ExperimentalWarning' ava"
```

Comment:
Node.js 20 changed how ESM loaders work, breaking AVA support for `"nodeArguments": ["--loader=tsx"]` configuration. This is a workaround, per https://github.com/avajs/ava/issues/2593#issuecomment-1524846453